index.css

```diff
@@ -0,0 +1,46 @@
+* {
+  box-sizing: border-box;
+}
+body {
+  margin: 0;
+  padding: 0;
+  font-weight: 300;
+  font: 300 1em/1.4 'Open Sans', sans-serif;
+  color: #404040;
```

Comment:
@mattgstevens had a good point in chat earlier that colors and other constants should be variables, so that they are reuseable